docs/source/api/python/nn-tensorflow.rst

```diff
@@ -42,4 +42,4 @@ Heterogeneous Learning Modules
     :nosignatures:
     :template: classtemplate.rst
 
-    ~dgl.nn.tensorflow.glob.HeteroGraphConv
+    ~dgl.nn.tensorflow.hetero.HeteroGraphConv
```

Comment:
I think `dgl.nn.tensorflow.HeteroGraphConv` should be enough as in https://github.com/dmlc/dgl/blob/master/docs/source/api/python/nn-pytorch.rst

Reply:
Sure. Let's use the same format as in the PyTorch doc.